specs/introduction.md

```diff
@@ -0,0 +1,57 @@
+# Introduction
+
+Sphinx is a protocol that aims to make the smart contract deployment process more secure, transparent, and efficient. This document provides an overview of Sphinx to serve as context for the rest of the specification.
+
+## Goals
+
+* **Secure**: Deployments with Sphinx are tamper-proof. This is crucial because of the mission-critical nature of smart contract deployments, where subtle changes in a deployment can cause significant issues after a system has been deployed.
+* **Transparent**: Sphinx offers greater transparency by providing a verifiable hash of each deployment. Users and other stakeholders can verify the transactions in a deployment before it's executed.
+* **Efficient**: Developers can approve arbitrarily large deployments across an arbitrary number of chains by signing the deployment's unique identifier using a meta transaction. Once it's approved, Sphinx's backend will trustlessly execute the deployment on every chain. Developers don't need native gas tokens on any chain.
+
+## Architecture Overview
+
+Most teams use a multisig wallet to approve permissioned actions. We wanted teams to be able to approve deployments from their multisig as well, so we built our protocol on top of Gnosis Safe. Our primary on-chain component is a [Safe Module](TODO(end)), which allows us to use our own signature verification mechanism that's designed for deployments (details below).
```

Comment:
I think the ordering of this section makes it difficult to understand. I also think it could be cleanly split into two sections. I would start with a high level `Architecture Overview` section, then followup with a `Merkle Tree Architecture`.

In the `Architecture Overview`, I would just explain at a high level how we intend to accomplish the stated goals. So something like: 

"To achieve the above goals Sphinx implements a custom transaction verification scheme that allows a set of multisig owners to approve a deployment including an arbitrarily large number of transactions across an arbitrary set of networks using an off chain meta transaction. Once a deployment is approved, it can be executed trustlessly by an external third party."

Then in the same section, I we should explain that we use a Merkle tree as an efficient data structure for storing all of the transactions. Finally, we should end this section by explaining that we implement this verification scheme as a module installed in a Gnosis Safe since that is the industry standard for multi-signature wallets. 

I'd then have a second section titled `Merkle Tree` which focuses specifically on the structure of the Merkle Tree. I think the content you have here related to the Merkle Tree is solid.